client/src/components/layout/ReviewDeleteButton.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+
+const ReviewDeleteButton = ({ user, review, getPlanet }) => {
+  
+  const deleteReview = async (event) => {
+    event.preventDefault()
+    const reviewBody = {
+      reviewId: review.id,
+    }
+    try {
+      const response = await fetch(`/api/v1/planets/${review.planetId}/reviews/delete-review`, {
+        method: "DELETE",
+        headers: new Headers({
+          "Content-Type": "application/json",
+        }),
+        body: JSON.stringify(reviewBody),
+      })
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw (error)
+      }
+      getPlanet()
+    } catch (error) {
+      console.error(error)
+    }
+  }
+
+  return(
+    <>
+      { user.id === review.userId ? <button className="delete-button" onClick={deleteReview}>Delete Review</button> : null }
+    </>
```

Comment:
Multiline ternaries are somewhat discouraged, but not full antipattern
https://github.com/airbnb/javascript#comparison--nested-ternaries

Either replace this with an if statement above the return to make this more readable, or at least follow the airbnb suggestion for multine

Reply:
this code may along not need the <> fragement tags

Reply:
I was getting errors without the fragments.